### PR TITLE
add documentation for DB customizations

### PR DESCRIPTION
**Pull request by 3nids**
corresponding to https://github.com/QGEP/datamodel/pull/95

### CODE REVIEW

File: en/admin-guide/db-customization/index.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+The combination of both allows to introduce small customizations of the data model such as adding fields to existing base tables or extending views by joining additional tables.
+
+
+This chapter describes these possible customizations.
+
+
+Adding fields
+^^^^^^^^^^^^^
+
+It is allowed to add additional fields to tables by naming them as ``XXX` (TO BE DEFINED).
```
Comment (lbartoletti) [inline en/admin-guide/db-customization/index.rst]: To be defined?

File: en/admin-guide/db-customization/index.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+If added in base views, these additional fields will be automatically added to the views, being an editable field. The views can be re-generated any time using the following command:
+
+``PGSERVICE=pg_qgep SRID=2056 ./view/create_views.py``
```
Comment (lbartoletti) [inline en/admin-guide/db-customization/index.rst]: I added a comment about it in the PR

File: en/admin-guide/db-customization/index.rst (hunk 1 above)
Comment (3nids) [reply en/admin-guide/db-customization/index.rst]: we should define a way of naming field (with `usr_*` or whatever)

File: en/admin-guide/db-customization/index.rst (hunk 2 above)
Comment (3nids) [reply en/admin-guide/db-customization/index.rst]: fixed

File: en/admin-guide/db-customization/index.rst (hunk 1 above)
Comment (ponceta) [reply en/admin-guide/db-customization/index.rst]: I think we should go to usr_* like QWAT. @sjib thoughts?

File: en/admin-guide/db-customization/index.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
+Customization of the data model
+===============================
+
+This chapter describes the possible customization to the data model.
+
```
Comment (ponceta) [inline en/admin-guide/db-customization/index.rst]: Please add :

"Before going further, we strongly recommend to ask the community if you have any doubts about how to store any information in the database.

If QGEP datamodel is not yet able to receive your data, please let us know what you want to store and how you would store it, QGEP model is a standard proof adaptative model. We want the standard to be as near as possible as your needs.

If none of the above solutions fit your needs, the following explanations and rules explain how to enhance QGEP for your specific needs."

Ping @sjib

**Comment by m-kuhn** [comment]
@sjib is currently out of office until next Monday, 13.5.

 * If the `usr_` prefix works well in practice and is the current state of art for the QGEP group, my vote to this is +1 to keep things aligned
 * Also the request for community feedback will be good to add as a note

**Comment by lbartoletti** [comment]
@3nids can you please update the docs with "usr_" and the suggestion from @ponceta ?

After that, it will be good for me.

Thanks

**Comment by 3nids** [comment]
done

File: en/admin-guide/db-customization/index.rst (hunk 3 above)
Comment (sjib) [reply en/admin-guide/db-customization/index.rst]: Ok. Maybe we can reference somewhere that QGEP complies with SIA 405 Waste Water and VSA-DSS datamodel of sia and VSA.

Comment (sjib) [reply en/admin-guide/db-customization/index.rst]: sia: Link http://www.sia.ch/405
VSA-DSS: Link http://dss.vsa.ch

Comment (3nids) [reply en/admin-guide/db-customization/index.rst]: done

**Pull request merged**